libraries/script-engine/src/ScriptEngine.cpp

```diff
@@ -2369,9 +2369,12 @@ void ScriptEngine::entityScriptContentAvailable(const EntityItemID& entityID, co
         QList<QString> safeURLPrefixes = { "file:///", "atp:", "cache:" };
         safeURLPrefixes += qEnvironmentVariable("EXTRA_WHITELIST").trimmed().split(QRegExp("\\s*,\\s*"), QString::SkipEmptyParts);
 
-        // IF WHITELIST IS DISABLED IN SETTINGS
-        bool whitelistEnabled = Setting::Handle<bool>("private/whitelistEnabled", true).get();
-        if (!whitelistEnabled) {
+        // ENTITY SCRIPT WHITELIST TOGGLE CHECK
+        Setting::Handle<bool> whitelistEnabled{"private/whitelistEnabled", false };
+        bool isWhitelistEnabled = whitelistEnabled.get();
+                
+        if (!isWhitelistEnabled) {
```

Comment:
```suggestion
        if (!whitelistEnabled.get()) {
```
(and get rid of isWhitelistEnabled above, and maybe remove the print on the line below